src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -421,25 +415,18 @@ def _get_install_info(cmd,
             password1=True, password2=True, resource_group_name=resource_group_name)
         credentials = LongRunningOperation(cmd.cli_ctx)(poller)
         sync_username = credentials.username
-        sync_password = {
-            "password1": credentials.passwords[0].value,
-            "password2": credentials.passwords[1].value
-        }
+        sync_password = credentials.passwords[0].value
```

Comment:
if we're going to only use password one. You need to make sure we only rotate password 1